src/languages/python-repl.js

```diff
@@ -0,0 +1,29 @@
+/*
+Language: Python REPL
+Requires: python.js
+Author: Josh Goebel <hello@joshgoebel.com>
+Category: common
+*/
+
+export default function(hljs) {
+  return {
+    aliases: ['pycon'],
+    contains: [
+      {
+        className: 'meta',
+        starts: {
+          // a space separates the REPL prefix from the actual code
+          // this is purely for cleaner HTML output
+          end: / /,
+          starts: {
+            end: '$', subLanguage: 'python'
+          }
+        },
+        variants: [
+          { begin: /^>>>(?=[ ])/ },
+          { begin: /^\.\.\.(?=[ ])/ }
```

Comment:
Python REPL accepts tabs (`\t`) as well in addition to spaces

Reply:
But this is the character between the REPL lines in the output... are you saying that is a space and not a tab?  We're not talking about what the interpreter supports here.

Reply:
I'm not even sure how I would test that... If I copy and paste it from terminal it'll be a space...

Reply:
Ohhh oops! Sorry about that, I misunderstood the purpose of that space. You are correct, those are spaces coming in from terminal

Reply:
I used to just throw \s everywhere but that can do very bad things sometimes when it starts matching line endings (CR/LF, etc) as white space.

Reply:
Should the space be optional? See the copy/paste from my local terminal as an example

```
Python 3.7.3 (default, Dec 23 2019, 21:25:06)
[Clang 11.0.0 (clang-1100.0.33.16)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> def hello():
...     print("hello")
...
>>> hello()
hello
>>>
>>>
```

Reply:
I'm ok with making it space or $... that would cover that edge case.